apps/src/code-studio/components/progress/StudentTable.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import i18n from '@cdo/locale';
+
+const styles = {
+  table: {
+    width: '90%',
+    margin: 'auto'
+  },
+  tr: {
+    height: 41,
+    color: color.cyan,
+    border: `1px solid ${color.lighter_cyan}`,
+    backgroundColor: color.lightest_gray,
+    ':hover': {
+      backgroundColor: color.lighter_cyan,
+      cursor: 'pointer'
+    }
+  },
+  td: {
+    padding: 10
+  },
+  selected: {
+    fontFamily: '"Gotham 7r", sans-serif',
+    color: color.white,
+    backgroundColor: color.light_cyan
+  }
+};
+
+export const studentShape = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired
+});
+
+class StudentTable extends React.Component {
```

Comment:
Great to see this in React! You're probably aware of this, but I just wanted to include a pointer to the haml this could eventually be used to replace: https://github.com/code-dot-org/code-dot-org/blob/37f78d64fe31128fbbcb3064eae84f73688ef69c/dashboard/app/views/levels/_teacher.html.haml#L100-L101